targets/nuttx-stm32f4/README.md

```diff
@@ -86,6 +86,25 @@ Followings are the options to set:
 * Enable `Application Configuration -> System Libraries and NSH Add-ons -> IoT.js`
 * Enable all childs of `Application Configuration -> System Libraries and NSH Add-ons -> readline() Support` (for those who wants to use readline)
 
+##### 3.1. For those who want to use `net` module
```

Comment:
It looks it is additional option for ``net`` module. 
I think it is part of ``Followings are the options to set:``, but its numbering  is ``3.1``.

How do you think above options for ``Common`` and make ``For net module``, like:
- Common
 * Change `Build Setup -> Build Host Platform` from _Windows_ to [_Linux_|_OSX_]
 * Enable `System Type -> FPU support`
...
- For ``net`` module
 * Enable `System Type -> STM32 Peripheral Support -> Ethernet MAC`
 * Disable `System Type -> STM32 Peripheral Support -> USART2`
...